package.json

```diff
@@ -47,8 +47,10 @@
         "@storybook/addon-viewport": "^8.6.11",
         "@storybook/react": "^8.6.11",
         "@storybook/react-vite": "^8.6.11",
-        "@swc-node/register": "^1.6.6",
-        "@swc/core": "^1.3.68",
+        "@swc-node/register": "^1.10.10",
+        "@swc/core": "1.11.13",
+        "@swc/helpers": "^0.5.15",
+        "@swc/jest": "^0.2.37",
```

Comment:
You don't seem to be importing these, I'm guessing it's just behind-the-scenes implicit magic somewhere?

Reply:
`@swc/jest` is used in `test.config` as a Jest `transform`. 
I've removed `@swc/helpers` (I think I added it while trying to get something working).